### PR TITLE
Fetch and render images from BE

### LINKED ISSUE #6: add action to frontend calling backend and rendering result

### PR DESCRIPTION
**Pull request by jCobbSK**
closes #6

### CODE REVIEW

File: client/src/store/Fetcher.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import axios from 'axios';
+
+export const Fetcher = axios.create({
+  baseURL: 'http://localhost:4000/api/v1/',
```
Comment (pejrak) [inline client/src/store/Fetcher.ts]: we should probably configure this, eventually...

File: client/src/store/reducers/ImagesReducer.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { AnyAction } from 'redux';
+import { LOAD_IMAGES } from '../actions';
+
+export interface Image {
+  id: number;
+  name: string;
```
Comment (pejrak) [inline client/src/store/reducers/ImagesReducer.ts]: name is not needed right now, let's remove stuff we don't need

Comment (pejrak) [reply client/src/store/reducers/ImagesReducer.ts]: can you replace the name with *fileName* ? I will add that

File: client/src/store/Fetcher.ts (hunk 1 above)
Comment (jCobbSK) [reply client/src/store/Fetcher.ts]: It's temporary until we'll have build setup.

File: client/src/store/reducers/ImagesReducer.ts (hunk 2 above)
Comment (jCobbSK) [reply client/src/store/reducers/ImagesReducer.ts]: Will change in upcoming PR so this is not blocked.

**Pull request merged**